xml/System.Threading/EventWaitHandleAcl.xml

```diff
@@ -41,14 +41,28 @@
         <Parameter Name="eventSecurity" Type="System.Security.AccessControl.EventWaitHandleSecurity" />
       </Parameters>
       <Docs>
-        <param name="initialState">To be added.</param>
-        <param name="mode">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="createdNew">To be added.</param>
-        <param name="eventSecurity">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="initialState">
+          <see langword="true" /> to set the initial state to signaled if the named event is created as a result of this call; <see langword="false" /> to set it to non-signaled.</param>
+        <param name="mode">One of the enum values that determines whether the event resets automatically or manually.</param>
+        <param name="name">The name, if the event is a system-wide synchronization event; otherwise, <see langword="null" /> or an empty string.</param>
+        <param name="createdNew">When this method returns, this argument is always set to <see langword="true" /> if a local event is created; that is, when <paramref name="name" /> is <see langword="null" /> or <see cref="F:System.String.Empty" />. If <paramref name="name" /> has a valid, non-empty value, this argument is set to <see langword="true" /> when the system event is created, or it is set to <see langword="false" /> if an existing system event is found with that name. This parameter is passed uninitialized.</param>
+        <param name="eventSecurity">The optional Windows access control security to apply.</param>
+        <summary>Gets or creates an <see cref="T:System.Threading.EventWaitHandle" /> instance, allowing a <see cref="T:System.Security.AccessControl.EventWaitHandleSecurity" /> instance to be optionally specified to set it during the event creation.</summary>
+        <returns>An object that represents a system event wait handle, if named, or a local event wait handle, if nameless.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+If a `name` is passed and the system event already exists, the existing event is returned. If `name` is `null` or <xref:System.String.Empty>, a new local event is always created.
+
+          ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">.NET Framework only: The <paramref name="name" /> length is beyond MAX_PATH (260 characters).</exception>
```

Comment:
The `MAX_PATH` issue would result in an `ArgumentException`, not an `ArgumentNullException`. From legacy code review anyway, I don't see a case where `ArgumentNullException` would be thrown by this method.

Reply:
Good catch.
```suggestion
        <exception cref="T:System.ArgumentException">.NET Framework only: The <paramref name="name" /> length is beyond MAX_PATH (260 characters).</exception>
```